src/redis/README.md

```diff
@@ -0,0 +1,27 @@
+# VIP Go Redis Helper Library
+
+## Initialization
+
+Instantiate a new Redis client:
+
+``` js
+const { redis } = require( '@automattic/vip-go' );
```

Comment:
I think we still need to expose `redis` in https://github.com/Automattic/vip-go-node/blob/master/src/index.js in order to be able to import it like this.

In addition, I see we are exporting a `new Redis()` here https://github.com/Automattic/vip-go-node/pull/125/files#diff-66b52dd10fa44ca40f44c06793a2aec1R77 but I guess in order to give the client the possibility to pass their own logger, we should just export the `Redis` class, and they need to do the `new Redis ( { logger: myLogger } );` on their own.

Or if we can avoid exporting classes, that would be good as all our other helpers (logger, newrelic, server) do not use a class approach when used by clients. It would be good to keep a consistency of:

```
const { logger, server, newrelic, redis } = require( '@automattic/vip-go-node' );

logger();
server();
newrelic();
redis();
```

We can use the class approach in the helper itself, but just exposing a function instead to keep the consistency. Hope I've been clear, if it's not the case, please let me know